client/templates/other/slack.html

```diff
@@ -36,24 +36,93 @@ <h3>Quick Links</h3>
+                    <p><b><i>We are a pay-it-forward community.</i></b> If someone here has helped you, please give back by helping someone else!</p>
+                    <p>.</p>
+                    <p>It doesn’t matter if you can’t answer a question in a brief sentence, or if you get pulled away before you can type out further 
+                      replies. If a question isn’t clear, ask a clarifying question. If you remember something you read that seemed related, send a 
+                      link. A lot of people who are stuck will be fine if you can’t provide the solution, they just need another place to look.</p>
+                  <hr>
+                  <h2><img src="/images/logo-circle.png" style="width:50px;height:50px">Be polite</h2>
+                    <p>All the CodeBuddies here deserve your respect, so be polite. Not “retail employee” polite......but don’t fan flamewars, get 
+                      sucked into lose-lose arguments, or other internet classics.</p>
+                  <hr>
+                  <h2><img src="/images/logo-circle.png" style="width:50px;height:50px">Be a model Slack citizen</h2>
+                    <p>Be the change you want to see in the world! <i>Remember ours runs on volunteers and collaboration.</i> If a conversation is 
+                      getting heated, try to defuse it. If a hard question is unanswered, dig in &amp; work through it with others. If a channel is full 
+                      of unanswered questions, mention in <b>#codebuddies-meta</b> that you’d like to take ownership of it.</p>
+                <hr>
+                <hr>
+                <p>Thanks to everyone who've been following these etiquette rules already! This community is <span style="color: blue; font-size: 20px"><i class="fas fa-globe-asia"></i></span> 
+                  <b><i>international and distributed across timezones</i></b>, and it's nice to find that even with our diverse backgrounds, experience levels, and 
+                  nationalities, we have similar goals and can support each other <span style="color: rgb(248, 121, 142); font-size: 20px"><i class="fas fa-heart"></i></span>. 
+                  It's always a delight to be late-night coding – &amp; suddenly realize that halfway around the world, a cheerful hangout collaborator is pouring their first cup of coffee for the day.</p>
+              </div>
               <h3>How do you show your appreciation to someone on Slack?</h3>
                 <p>We use a bot called plusplus to applaud or encourage each other. For example, if @hermionegranger gives me a helpful answer to a question I asked in a channel, I might type @hermionegranger++.
-                  Think of these points as virtual cookies. :)</p>
+                  Think of these points as virtual cookies. <span style="font-size: 20px"><i class="far fa-smile-wink"></i></span></p>
               <h3>Who can create new channels on Slack?</h3>
```

Comment:
Sorry I didn't mention it earlier: we can delete these entire question and answer ("Who can create new channels on Slack?") because now anybody can create a public channel again, and the public channel will be posted automatically to #announcements so admins can track it and know if we have a duplicate channel, or something.